libuv-demo/src/main/cpp/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.10)
+
+set(LIBUV_VERSION v1.38.0)
+
+include(ExternalProject)
+
+# https://stackoverflow.com/questions/48132569/cmake-externalproject-superbuild-isnt-executed-with-android-ndk
+# Forward Android CMake cross compile args to ExternalProject.
+set(ANDROID_CMAKE_ARGS
+        -DANDROID_ABI=${ANDROID_ABI}
+        -DANDROID_PLATFORM=${ANDROID_PLATFORM}
+        -DCMAKE_LIBRARY_OUTPUT_DIRECTORY=${CMAKE_LIBRARY_OUTPUT_DIRECTORY}
+        -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
+        -DANDROID_NDK=${ANDROID_NDK}
+        -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
+        -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}
+        -DCMAKE_MAKE_PROGRAM=${CMAKE_MAKE_PROGRAM}
+        -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
+        -G${CMAKE_GENERATOR})
+
+ExternalProject_Add(libuv
```

Comment:
Looks like first time not auto download this ? What do I missing ? @bnoordhuis

Reply:
For now, I have to remove ${libuv_static} from last line, and libuv is download, add it back.

Reply:
Turns out this is related to ninja.